src/index.html

```diff
@@ -17,6 +17,54 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__item">
+            <a
+              class="is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              data-qa="hover"
```

Comment:
The `data-qa` attribute is used here, which is typically for testing purposes. Ensure this is intentional and necessary for your project.